docs/oscar_references.bib

```diff
@@ -2244,6 +2244,19 @@ @Book{Loo84
   doi           = {10.1017/CBO9780511662720}
 }
 
+@InCollection{Los18,
+  author        = {Losev, Ivan},
+  title         = {Procesi bundles and symplectic reflection algebras},
+  booktitle     = {Algebraic and analytic microlocal analysis},
+  mrnumber      = {3903316},
+  series        = {Springer Proc. Math. Stat.},
+  volume        = {269},
+  publisher     = {Springer, Cham},
+  pages         = {3--61},
+  year          = {2018},
+  doi           = {10.1007/978-3-030-01588-6\_1}
```

Comment:
remind to self to verify how escaping works in dois (no action from your side required)